src/api/emojiReactionAdd.js

```diff
@@ -9,5 +10,6 @@ export default (
   apiPut(
     auth,
     `messages/${messageId}/emoji_reactions/${emojiName}`,
+    {},
```

Comment:
I added empty object here because it looked that function was passing as a parameter object.